docs/examples/frameworks/jax/flax-basic_example.ipynb

```diff
@@ -9,14 +9,14 @@
     "\n",
```

Comment:
<p>(...) creating ~a~<strong>an</strong> iterator definition function. It is <strong>a </strong>slightly modified (...)</p><p> </p><p><em>Simpler / less verbose version:</em></p><p><s>Note t</s><strong>T</strong>he new arguments passed to <code>fn.readers.caffe2</code><s>:</s><strong>,</strong> <em>(comma instead of a colon) </em><code>num_shards</code> and <code>shard_id~</code>. They are used to~<strong>,</strong> <em>(comma)</em> control sharding:</p><p> </p><p>We add <code>devices</code> argument to the decorator to <s>indicate to the iterator</s> <strong>specify</strong> which devices we want to use. (...)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/5205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Done with some changes.</p>